README.md

```diff
@@ -33,7 +33,7 @@ cass-operator-555577b9f8-zgx6j   1/1     Running   0          25h
 
 ### Creating a storage class
 
-You will need to create an appropriate storage class which will define the type of storage to use for Cassandra nodes in a cluster. For example, here is a storage class for using SSDs in GKE, which you can also find at [operator/deploy/k8s-flavors/gke/storage.yaml](operator/k8s-flavors/gke/storage.yaml):
+You will need to create an appropriate storage class which will define the type of storage to use for Cassandra nodes in a cluster. For example, here is a storage class for using SSDs in GKE, which you can also find at [operator/deploy/k8s-flavors/gke/storage.yaml](https://github.com/datastax/cass-operator/blob/master/operator/k8s-flavors/gke/storage.yaml):
```

Comment:
does github have any kind of templating magic to make this work better on a branch/tag/hash?